### ISSUE TITLE
Error with request: name 'bytes_sent' is not defined

### ISSUE THREAD

**Issue by skickar** [deckerego/ampule#16]
I'm trying to send an HTML file converted to a string as a response, like this:

```
def getHTML(filepath):
    f = open(filepath,'r', encoding="utf-8")
    html = f.read()
    f.close()
    return html

@ampule.route("/")
def light_set(request):
    return (200, {}, getHTML('cat.html'))
```

When I exceed about 50 lines, I get "Error with request: name 'bytes_sent' is not defined" 

It looks like it's referring to this line:

  ```
  while True:
        try:
            bytes_sent_total += client.send(response)
            if bytes_sent_total >= response_length:
                return bytes_sent_total
            else:
                response = response[bytes_sent:]
                continue
        except OSError as e:
            if e.errno == 11:       # EAGAIN: no bytes have been transfered
                continue
            else:
                return bytes_sent_total
```

I can't seem to send a file over a certain size without tripping this.

**Comment by deckerego** [comment]
@skickar will get a fix out today

**Comment by deckerego** [comment]
Fixed in Release 0.2.1
